{"primaryContentSections":[{"declarations":[{"languages":["occ"],"platforms":["iOS"],"tokens":[{"text":"MLNErrorCodeUnknown","kind":"identifier"}]}],"kind":"declarations"}],"hierarchy":{"paths":[["doc:\/\/org.swift.MyProject\/documentation\/MapLibre","doc:\/\/org.swift.MyProject\/documentation\/MapLibre\/MLNErrorCode"]]},"identifier":{"url":"doc:\/\/org.swift.MyProject\/documentation\/MapLibre\/MLNErrorCode\/MLNErrorCodeUnknown","interfaceLanguage":"occ"},"variants":[{"traits":[{"interfaceLanguage":"occ"}],"paths":["\/documentation\/maplibre\/mlnerrorcode\/mlnerrorcodeunknown"]}],"schemaVersion":{"major":0,"patch":0,"minor":3},"sections":[],"kind":"symbol","metadata":{"role":"symbol","navigatorTitle":[{"kind":"identifier","text":"MLNErrorCodeUnknown"}],"modules":[{"name":"MapLibre"}],"title":"MLNErrorCodeUnknown","symbolKind":"case","externalID":"c:@E@MLNErrorCode@MLNErrorCodeUnknown","fragments":[{"kind":"identifier","text":"MLNErrorCodeUnknown"}],"roleHeading":"Enumeration Case"},"abstract":[{"type":"text","text":"An unknown error occurred."}],"references":{"doc://org.swift.MyProject/documentation/MapLibre/MLNErrorCode":{"type":"topic","abstract":[{"type":"text","text":"Error constants for the Mapbox SDK."}],"role":"symbol","fragments":[{"text":"MLNErrorCode","kind":"identifier"}],"kind":"symbol","identifier":"doc:\/\/org.swift.MyProject\/documentation\/MapLibre\/MLNErrorCode","url":"\/documentation\/maplibre\/mlnerrorcode","title":"MLNErrorCode"},"doc://org.swift.MyProject/documentation/MapLibre":{"abstract":[{"text":"Powerful, free and open-source mapping toolkit with full control over data sources and styling.","type":"text"}],"role":"collection","type":"topic","kind":"symbol","url":"\/documentation\/maplibre","title":"MapLibre","identifier":"doc:\/\/org.swift.MyProject\/documentation\/MapLibre"},"doc://org.swift.MyProject/documentation/MapLibre/MLNErrorCode/MLNErrorCodeUnknown":{"role":"symbol","kind":"symbol","identifier":"doc:\/\/org.swift.MyProject\/documentation\/MapLibre\/MLNErrorCode\/MLNErrorCodeUnknown","url":"\/documentation\/maplibre\/mlnerrorcode\/mlnerrorcodeunknown","title":"MLNErrorCodeUnknown","fragments":[{"kind":"identifier","text":"MLNErrorCodeUnknown"}],"type":"topic","abstract":[{"type":"text","text":"An unknown error occurred."}]}}}